java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -230,6 +278,7 @@ public int hashCode() {
       flowBuilder.addAll(laFlow);
 
       PSet<Symbol> newTrackSymbols = newTrackedSymbols(edge);
+      SameConstraints newSameConstaints = new SameConstraints(sameConstraints, newTrackSymbols, domains);
```

Comment:
Typo in variable name.  You can test if there is any change in `trackedSymbols` and create instance only if necessary. It will be often the case that symbols won't be changed.